src/__tests__/opportunities.int.test.ts

```diff
@@ -118,15 +118,18 @@ describe('/opportunities', () => {
         RETURNING id, title;
       `);
       logger.debug('opportunityIds: %o', opportunityIds);
-      const sparkleOpportunityId = opportunityIds.rows
-        .filter((it) => it.title === '✨')[0].id;
-      logger.debug('sparkleOpportunityId: %d', sparkleOpportunityId);
+      const sparkleOpportunity = opportunityIds.rows
+        .filter((it) => it.title === '✨')[0];
+      if (sparkleOpportunity === undefined) {
```

Comment:
This would help in the situation where no rows came back at all, right? Then it would fail with the below message rather than attempting to get a property on undefined? If so, yes, this is much better.

Reply:
Correct!